### PR TITLE
update to python3

### PR DESCRIPTION
**Pull request by eyalpalo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Upgraded to python3 and deprecated

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Mimecast/Scripts/MimecastFindEmail/MimecastFindEmail.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+MAILBOXES_CTXKEY = "Mailboxes"
+
+
+def main() -> None:
+    res = []
+    resp = demisto.executeCommand("mimecast-query", demisto.args())
+    users = set()
```
Comment (tkatzir) [inline Packs/Mimecast/Scripts/MimecastFindEmail/MimecastFindEmail.py:10]: No need to define it here (too early, too much context...)
Move the definition to inside the `if data`

File: Packs/Mimecast/Scripts/MimecastFindEmail/MimecastFindEmail_test.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+def test_mimecast_find_email(mocker):
+    mocker.patch.object(demisto, 'executeCommand', return_value=QUERY_COMMAND_RESPONSE)
+    mocker.patch.object(demisto, 'results')
+    main()
+    results = demisto.results.call_args[0][0]
+
+    assert '### Mailboxes with email(s) matching the query:' in results[0].get('Contents')
+    assert results[1].get('Contents') == 'yes'
```
Comment (tkatzir) [inline Packs/Mimecast/Scripts/MimecastFindEmail/MimecastFindEmail_test.py:23]: Missing a test for no `data`, with a check for `'no'`.

**Review by tkatzir — COMMENTED** [review]
Looks good, some minor comments.

**Review by tkatzir — APPROVED** [review]

**Comment by eyalpalo** [comment]
Need to be force merged because script name fails the build (starting with script-xxx, no need to change it, this script is deprecated).

**Pull request merged**